### ISSUE TITLE
VGG16 convergence comparison on CPU between Fluid and TensorFlow

### ISSUE THREAD

**Issue by kuke** [PaddlePaddle/Paddle#6944]
![vgg_converge_on_cpu](https://user-images.githubusercontent.com/3064195/34327252-3bccc330-e8fa-11e7-81df-688639a8737e.png)

- Data set: cifar10
- Epochs: 50
- Batch size: 128
- Optimizer: Adam
- Learning rate = 1e-3
